salt/modules/cmdmod.py

```diff
@@ -3184,9 +3185,14 @@ def powershell(cmd,
         python_shell = True
 
     # Append PowerShell Object formatting
-    cmd += ' | ConvertTo-JSON'
-    if depth is not None:
-        cmd += ' -Depth {0}'.format(depth)
+    # ConvertTo-JSON is only available on Versions of Windows greater than
```

Comment:
Would have thought ConvertTo-JSON is PowerShell Version dependant.

Reply:
You're right... should use the shell_info function to get the version of PowerShell that's installed...